src/TestUtils/src/Microsoft.Maui.IntegrationTests/BaseBuildTest.cs

```diff
@@ -98,25 +98,10 @@ public void BuildTestSetUp()
 		[TearDown]
 		public void BuildTestTearDown()
 		{
-			// Clean up test or attach content from failed tests
```

Comment:
I deleted the code that would clean up test artifacts if the test passed. As it turned out, tests were passing despite there being bugs in the product, but successful tests had no logs. With this change, every template test has binlogs and console output making it a lot easier to understand why a test passed even though perhaps it should have failed.

Reply:
What happens if we re-run ? this would previous delete the previous results. My issue here is if they can get mixed up if have similar names. Do you know if we add the DateTime stamp to these logs?

Reply:
Rerun I think is fine because it presumably always gets a clean instance, no? I assume that because there are all kinds of ways test can fail without cleanup running and I'd _hope_ that wouldn't cause a rerun to fail?